docs/concepts/events-and-resources.md

```diff
@@ -0,0 +1,108 @@
+---
+description: Events and resources are the primary units of observability in Prefect Cloud
+tags:
+    - resources
+    - events
+    - observability
+---
+
+# Events
+
+Events are the primary unit of observability in Prefect Cloud, and are designed to be a simple but complete record of something occuring somewhere in your stack at a given time. Prefect events can represent API calls, state changes, or events from your execution environment or infrastructure. Events enable both analysis of your data stack via (event feeds[link]) and the confiuration of stack reactivity via (automations[link])
```

Comment:
"Events and resources are the primary units of observability..."
"Events are the primary unit of observability..."

Should these be consistent?